newIDE/app/scripts/import-GDJS-Runtime.js

```diff
@@ -53,6 +53,7 @@ if (!args['skip-sources']) {
 
     const startTime = Date.now();
 
+    const pixiDestinationPath = path.join(destinationPath, 'Runtime-sources', 'pixi');
```

Comment:
Should `pixi` and `three` be a sub-folder of `libs`?

Reply:
I would at least make a "types" folder so that it's a bit clearer for those looking at the runtime source files - should be obvious that they won't find actual source files and that it's probably imported from somewhere.